glue-codes/python/pyOpenFAST/interface_abc.py

```diff
@@ -30,7 +30,7 @@ class OpenFASTInterfaceType(CDLL):
     #           ErrMsgLen variable in the NWTC_Base.f90 file.  If that ever
     #           changes, it may be necessary to update the corresponding size
     #           here.
-    ERROR_MSG_C_LEN = 1025
+    ERROR_MSG_C_LEN = 8196
```

Comment:
We had this at `ErrMsgLen+1` before to accommodate the appending a `C_NULL_CHAR` after a full message of length `ErrMsgLen`.  While unlikely we would ever have a full message buffer now, does the logic for the F to C message conversion handle stripping off a character to make space for the `C_NULL_CHAR`?

Reply:
Good catch, fixed